src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -312,12 +312,32 @@ def _flexible_server_params(command_group):
                  'To learn more about IOPS based on compute and storage, refer to IOPS in Azure Database for MySQL Flexible Server'
         )
 
+        iops_v2_arg_type = CLIArgumentType(
+            type=int,
+            options_list=['--iops'],
+            help='Value of IOPS in (operations/sec) to be allocated for this server. '
+                 'This value can only be updated if flexible server is using Premium SSD v2 Disks.'
+        )
+
+        throughput_arg_type = CLIArgumentType(
+            type=int,
+            options_list=['--throughput'],
+            help='Storage throughput in (MB/sec) for the server. '
+                 'This value can only be updated if flexible server is using Premium SSD v2 Disks.'
+        )
+
         auto_grow_arg_type = CLIArgumentType(
             arg_type=get_enum_type(['Enabled', 'Disabled']),
             options_list=['--storage-auto-grow'],
             help='Enable or disable autogrow of the storage. Default value is Enabled.'
         )
 
+        storage_type_arg_type = CLIArgumentType(
+            arg_type=get_enum_type(['PremiumV2_LRS']),
+            options_list=['--storage-type'],
```

Comment:
i think we can allow users to pass storage type of v1 too and support the same. Or explicitly call out that in case you need SSDv1 you do not need to pass storage type